deltacat/compute/compactor/compaction_session.py

```diff
@@ -172,35 +166,26 @@ def _execute_compaction_round(
     # sort primary keys to produce the same pk digest regardless of input order
     primary_keys = sorted(primary_keys)
 
-    # collect cluster resource stats
-    # cluster_resources = ray.cluster_resources()
-    # logger.info(f"Total cluster resources: {cluster_resources}")
-    # logger.info(f"Available cluster resources: {ray.available_resources()}")
-    # cluster_cpus = int(cluster_resources["CPU"])
-    # logger.info(f"Total cluster CPUs: {cluster_cpus}")
-
-    # collect node group resources
-
     cluster_resources = ray.cluster_resources()
     logger.info(f"Total cluster resources: {cluster_resources}")
     if pg_config: # use resource in each placement group
-        node_resource_keys=None
         cluster_resources = pg_config[1]
```

Comment:
The implicit assumption that `pg_config[1]` is `cluster_resources`, and `pg_config[0]` is remote options makes it difficult to understand the code. Can we either document as method docstrings or create a separate class representing the data structure?

Reply:
makes sense.